tests/bfd/conftest.py

```diff
@@ -47,8 +48,8 @@ def bfd_cleanup_db(
     )
     for dut in duts:
         assert wait_until(
-            45, 2, 0, check_orch_cpu_utilization, dut, orch_cpu_threshold
-        ), "Orch CPU utilization {} > orch cpu threshold {} before starting the test".format(
+            120, 4, 0, check_orch_cpu_utilization, dut, orch_cpu_threshold
+        ), "Orch CPU utilization {} > orch cpu threshold {} after the test".format(
```

Comment:
I found that it (especially the RP) will sometimes take more time to settle down after rebooting or config reloading, so I bumped the timeout to 120s here.